www/src/components/searchbar-body.js

```diff
@@ -0,0 +1,184 @@
+import React, { Component } from 'react'
+import { InstantSearch, Hits, SearchBox, Stats, RefinementList } from 'react-instantsearch/dom'
+import distanceInWords from 'date-fns/distance_in_words'
+import presets from "../utils/presets"
+import Link from 'gatsby-link'
+
+import typography, { rhythm } from "../utils/typography"
+
+const Search = () => {
+  return (
+    <div className="container">
+      <div css={{
+        display: `flex`,
+        justifyContent: `center`,
+      }}>
+      <SearchBox
+        translations={{placeholder: 'Search Gatsby Library by keyword'}}
+      />
+      </div>
+      <div css={{
+        display: `none`
+      }}>
+        <RefinementList attributeName="keywords"
+          defaultRefinement={["gatsby-plugin"]}
+        />
+      </div>
+     <div css={{
+       paddingBottom: rhythm(1),
+     }}>
+       <Stats
+         translations={{
+           stats: function(n, ms){
+             return `${n} results`
+           }
+         }}
+       />
+     </div>
+     <div css={{
+       borderTop: `2px solid #F5F3F7`,
+       borderBottom: `2px solid #F5F3F7`,
+       borderLeft: `2px solid #F5F3F7`,
+     }}>
+      <div css={{
+        backgroundColor: `white`,
+        height: `calc(100vh - 275px)`,
+        border: `2 px solid red`,
+        overflowY: `scroll`,
+        WebkitOverflowScrolling: `touch`,
+        "::-webkit-scrollbar": {
+          width: `6px`,
+          height: `6px`,
+        },
+        "::-webkit-scrollbar-thumb": {
+          background: presets.lightPurple,
+        },
+        "::-webkit-scrollbar-track": {
+          background: presets.brandLighter,
+        },
+      }}>
+        <Hits hitComponent={Result} />
+      </div>
+    </div>
+      <div>
+        <h3 css={{
+          fontSize: rhythm(.75),
+          textAlign: `center`,
+        }}>Search by <Link to={`https://www.algolia.com/`} style={{color: `#744C9E`, border: `none`, boxShadow: `none`}}>Algolia</Link></h3>
```

Comment:
This looks great, but `Link to` should be `a href` here because external link, currently appends to URL at https://deploy-preview-3456--gatsbyjs.netlify.com/packages/

Reply:
Yeah, should be `<a href="https://www.algolia.com"...`